components/Players/DraggableQueue.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-list>
+    <draggable v-model="queue" v-bind="dragOptions" class="list-group">
+      <v-hover
```

Comment:
The usability of this is a bit confusing.

There's a grab handle on the left when hovering, which makes sense, but you can click and hold anywhere to move it, yet a single click plays the item.

Not sure how to make it make sense, but as-is it feels weird.

Reply:
Spotify doesn't have a handle and everybody understand how it works. It's nice to have but imo we're good without it, people is capable of figuring that stuff themselves.